### PR TITLE
Hugo note

### PR DESCRIPTION
**Pull request by maelle**

### CODE REVIEW

File: content/technotes/2019-01-10-hugo.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+date: '2019-01-10'
+topicid:
+authors:
+  - Maëlle Salmon
+categories: technotes
+tags:
+  - hugo
+---
+
+A bit more than one year ago, rOpenSci [launched its new website design](https://twitter.com/rOpenSci/status/925036483383148545), by the designer [Maru Lango](https://www.marulango.com/). Not only did the website appearance change (for the best!), but the underlying framework too. ropensci.org is powered by [Hugo](https://gohugo.io/), like `blogdown`! Over the last few months, we've made the best of this framework, hopefully improving your browsing experience (and trapping you into binge reading). In this note, we'll go over the main developments, as well as give some Hugo tips.
```
Comment (richfitz) [inline content/technotes/2019-01-10-hugo.md]: "for the best" :arrow_right: "for the better"

Comment (richfitz) [reply content/technotes/2019-01-10-hugo.md]: `ropensci.org` does not come through as a link

Comment (richfitz) [reply content/technotes/2019-01-10-hugo.md]: link `blogdown` to CRAN page?

File: content/technotes/2019-01-10-hugo.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+  - hugo
+---
+
+A bit more than one year ago, rOpenSci [launched its new website design](https://twitter.com/rOpenSci/status/925036483383148545), by the designer [Maru Lango](https://www.marulango.com/). Not only did the website appearance change (for the best!), but the underlying framework too. ropensci.org is powered by [Hugo](https://gohugo.io/), like `blogdown`! Over the last few months, we've made the best of this framework, hopefully improving your browsing experience (and trapping you into binge reading). In this note, we'll go over the main developments, as well as give some Hugo tips. 
+
+# Our website and its source
+
+Skip this section if you just want to know what's become better on our website!
+
+Our website is located at [ropensci.org](https://ropensci.org). Its source lives on GitHub in [the ropensci/roweb2 repo](https://github.com/ropensci/roweb2/). The website is deployed on Netlify from the master branch, and each pull request gets a Netlify preview link too, which is quite handy when reviewing new blog posts.
```
Comment (richfitz) [inline content/technotes/2019-01-10-hugo.md]: "deployed on Netlify" is not clear - does that mean they're the people who host it (rather the gh pages), the place it's built, or somrthing else?

File: content/technotes/2019-01-10-hugo.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+Nearly all the changes mentioned in this post were made to the _theme_ of our website which you can find [at this folder of the GitHub repo](https://github.com/ropensci/roweb2/tree/master/themes/ropensci/layouts). When modifying Hugo templates, [reading Hugo docs](https://gohugo.io/documentation/) is very helpful. When things go wrong (why doesn't this partial know about the author list?!) head to the [debugging template docs](https://gohugo.io/templates/template-debugging/). And if you can't figure out a way out, Hugo has got a [Discourse forum](https://discourse.gohugo.io/) like [all the cool kids these days](https://discuss.ropensci.org/). Needless to say, another useful tip when learning Hugo templating is to start small, with either a minimal theme and tiny website, or small updates, then your skills will improve.
+
+Other changes were adjustments to content such as YAML reformatting. Fear not, we rarely had to manually update metadata, we wrote scripts using the [`yaml` R package](https://cran.r-project.org/web/packages/yaml/index.html). Find [such a script](https://github.com/ropensci/roweb2/issues/197#issuecomment-394264824).
```
Comment (richfitz) [inline content/technotes/2019-01-10-hugo.md]: Not a canonical CRAN link :joy:

File: content/technotes/2019-01-10-hugo.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+Our website is located at [ropensci.org](https://ropensci.org). Its source lives on GitHub in [the ropensci/roweb2 repo](https://github.com/ropensci/roweb2/). The website is deployed on Netlify from the master branch, and each pull request gets a Netlify preview link too, which is quite handy when reviewing new blog posts.
+
+Nearly all the changes mentioned in this post were made to the _theme_ of our website which you can find [at this folder of the GitHub repo](https://github.com/ropensci/roweb2/tree/master/themes/ropensci/layouts). When modifying Hugo templates, [reading Hugo docs](https://gohugo.io/documentation/) is very helpful. When things go wrong (why doesn't this partial know about the author list?!) head to the [debugging template docs](https://gohugo.io/templates/template-debugging/). And if you can't figure out a way out, Hugo has got a [Discourse forum](https://discourse.gohugo.io/) like [all the cool kids these days](https://discuss.ropensci.org/). Needless to say, another useful tip when learning Hugo templating is to start small, with either a minimal theme and tiny website, or small updates, then your skills will improve.
```
Comment (richfitz) [inline content/technotes/2019-01-10-hugo.md]: While on theme, IMO the space between `<h2>` and the following `<p>` looks a bit compressed

File: content/technotes/2019-01-10-hugo.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+
+# More entry points to content
+
+On top of a few general pages, our website is the home of many [blog posts](/blog), [tech notes](/technotes) and [community calls](/commcalls). We've created and improved pages referencing content over all these categories, so that you might more easily bounce [from say one software peer review post to the next](/tags/software-peer-review), read [all posts by our community manager](/authors/stefanie-butland), or simply find [all the things we ever published](/archive).
+
+## Tags and authors pages
+
+We now have a [tags page](/tags) referencing all tags along with the number of posts associated to them. The tags of a blog post, tech note or comm call are shown at the bottom, so that after reading you can choose to check out more content on the topic. After tags became even more valuable, our community manager Stefanie Butland added many of them and for making that process smoother we've created a Shiny app to update tags more easily over many posts, [`blogyaml`](https://github.com/ropenscilabs/blogyaml). It's worked well for us, and if you notice something wrong, well it has an issue tracker!
```
Comment (richfitz) [inline content/technotes/2019-01-10-hugo.md]: link to the issue tracker

Comment (richfitz) [reply content/technotes/2019-01-10-hugo.md]: > created a Shiny app

Is this private?

File: content/technotes/2019-01-10-hugo.md [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+
+We also have added an [authors page](/authors) that's very similar. On the page of each author, you'll find their title for rOpenSci staff members, their online home, possibly their Twitter handle, and all the content they've authored on our website. [See for instance Stefanie Butland's author page](authors/stefanie-butland). These pages are linked from the bottom of posts for returning authors. Thanks to [Angela Li](https://angela-li.github.io/) for [requesting we added authors pages](https://github.com/ropensci/roweb2/issues/365).
```
Comment (richfitz) [inline content/technotes/2019-01-10-hugo.md]: "On the page of each author" :arrow_right: "on each author's page"

File: content/technotes/2019-01-10-hugo.md [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+
+Technically, getting tags and authors page and tag pages is quite straightforward with Hugo thanks to _taxonomies_. To switch them on, we added these lines to the website config.toml
+
+```
+[taxonomies]
+  tag = "tags"
+  author = "authors"
+```
+
+For authors, a challenge was moving metadata around, from storing authors' metadata (online home, Twitter username) inside posts metadata to having authors metadata files under content/authors, but as mentioned earlier, with the `yaml` package you can edit hundreds of Markdown files at once, granted you check the edits carefully.
```
Comment (richfitz) [inline content/technotes/2019-01-10-hugo.md]: Perhaps link to an example metadata file to make this clearer?

File: content/technotes/2019-01-10-hugo.md [hunk 8]
```diff
@@ -0,0 +1,101 @@
...
+
+## Archive
+
+With precious content scattered over three categories (blog, tech notes, community calls), getting an overview can be tricky hence our adding an [archive page](/archive) featuring all our website content since the beginning of times. Behind the scenes, we [list pages with a date](https://github.com/ropensci/roweb2/blob/master/themes/ropensci/layouts/_default/single.html#L29) and [feeds them to an archive partial](https://github.com/ropensci/roweb2/blob/master/themes/ropensci/layouts/partials/archive.html) that we use for the [technotes page](/technotes) and authors pages as well for consistent presentation.
```
Comment (richfitz) [inline content/technotes/2019-01-10-hugo.md]: beginning of times = really only a year ago on the relaunch?

File: content/technotes/2019-01-10-hugo.md [hunk 9]
```diff
@@ -0,0 +1,101 @@
...
+
+And should you ever get lost, we've now a [cool 404 page](/404), although mentioning it probably lessens its coolness.
+
+# More content
+
+In the previous section we mention the [community calls living in ropensci.org](/commcalls). They used to have their own source and subdomain, but we've transferred them which has several advantages:
+
+* When reading about community calls, you're looking at the same design than when reading tech notes which makes us happy since we like our website. Besides, thanks to the footer and header, you're more prone to bouncing on over parts of the website. The [community calls list](/commcalls) list and [individual pages](https://ropensci.org/commcalls/2018-12-18/) use partials from the rest of the website, and our Hugo experience. In each comm call page, the video recording is for instance embedded!
```
Comment (richfitz) [inline content/technotes/2019-01-10-hugo.md]: "comm call" :arrow_right:  "community call"

File: content/technotes/2019-01-10-hugo.md [hunk 10]
```diff
@@ -0,0 +1,101 @@
...
+
+* Community calls now being similar to blog posts and tech notes, they're cross-listed via their tags and authors so are more likely to be found.
```
Comment (richfitz) [inline content/technotes/2019-01-10-hugo.md]: :arrow_right: "Community calls are now more similar to blog posts and tech notes, and are cross-listed via their tags and authors so that they're more easy to find"

File: content/technotes/2019-01-10-hugo.md [hunk 11]
```diff
@@ -0,0 +1,101 @@
...
+
+# Feed updates
+
+If you want to keep up with our production, you can subscribe to different feeds (or [our newsletter](https://news.ropensci.org/) since it includes mentions of recent posts!).
+
+## All the feeds
+
+Our website has both RSS and JSON feeds. We added a JSON feed because we could and because some people might like that.
```
Comment (richfitz) [inline content/technotes/2019-01-10-hugo.md]: link to a technology that uses a json feed?

File: content/technotes/2019-01-10-hugo.md [hunk 12]
```diff
@@ -0,0 +1,101 @@
...
+
+# Twitter friendliness
+
+We're aware that our content does not only get shared on Twitter, but this social media platform is an important player so we've upped our Twitter game with two edits.
+
+{{< tweet 1073519094958313472>}}
+
+* We've worked on adding Twitter metadata to our [head partial](https://github.com/ropensci/roweb2/blob/master/themes/ropensci/layouts/partials/head.html) so when tweeting links from the website you get a pretty card (see above). We've used [resources listed in this GitHub issue](https://github.com/ropensci/roweb2/issues/290#issue-367542204) and [Twitter card validator](https://cards-dev.twitter.com/validator) on Netlify preview links. Whilst updating the head partial we realized not all our pages used it so we homogeneized all pages. Some of our pages like [about](/about) have fancy stuff that's not Markdown based, but we now handle that with partials listed in YAML and then called from the [single.html default template](https://github.com/ropensci/roweb2/blob/master/themes/ropensci/layouts/_default/single.html). Here [the source of our about page](https://github.com/ropensci/roweb2/blob/master/content/about.md).
```
Comment (richfitz) [inline content/technotes/2019-01-10-hugo.md]: This paragraph has ended up with different linespacing somehow

File: content/technotes/2019-01-10-hugo.md [hunk 13]
```diff
@@ -0,0 +1,101 @@
...
+
+## Not too much content for R-Bloggers
+
+R-Bloggers feed https://ropensci.org/rbloggers/index.xml
+
+We created an [R-Bloggers page which only outputs XML](https://github.com/ropensci/roweb2/blob/master/content/rbloggers/_index.md), with [an XML template](https://github.com/ropensci/roweb2/blob/master/themes/ropensci/layouts/rbloggers/rss.xml) ensuring this RSS feed is a full RSS feed contrary to the rest of our website that uses Hugo default RSS feeds with excerpts. We do not filter posts by category which means this not-so-R post (but still a bit R, thanks to our explaining how to browse our R stuff ;o) will appear in R-Bloggers, but we might do that later on.
+
+# Twitter friendliness
+
+We're aware that our content does not only get shared on Twitter, but this social media platform is an important player so we've upped our Twitter game with two edits.
```
Comment (richfitz) [inline content/technotes/2019-01-10-hugo.md]: "We're aware that our content does not only get shared on Twitter"  is hard to parse "while twitter is not the only platform on which our content is shared, ..."

**Comment by maelle** [comment]
Thanks a lot @richfitz! :smiley_cat:

File: content/technotes/2019-01-10-hugo.md (hunk 3 above)
Comment (maelle) [reply content/technotes/2019-01-10-hugo.md]: :joy: fixed

File: content/technotes/2019-01-10-hugo.md (hunk 4 above)
Comment (maelle) [reply content/technotes/2019-01-10-hugo.md]: I've changed level headers, but not sure it looks better now. Headings have been a problem for a while around here. cf #30

File: content/technotes/2019-01-10-hugo.md (hunk 5 above)
Comment (maelle) [reply content/technotes/2019-01-10-hugo.md]: I've made the link more obvious. It's not private, it's a public package that only we use :-)

File: content/technotes/2019-01-10-hugo.md (hunk 11 above)
Comment (maelle) [reply content/technotes/2019-01-10-hugo.md]: Since JSON feeds are so new, I've found nothing, but I've added a link to the official announcement. I've found a few enthusiastic blog posts but they were written right after the announcement.

Comment (maelle) [reply content/technotes/2019-01-10-hugo.md]: Maybe the format will never get adopted widely.

File: content/technotes/2019-01-10-hugo.md (hunk 12 above)
Comment (maelle) [reply content/technotes/2019-01-10-hugo.md]: I've removed the itemize here, and opened an issue about this #380

File: content/technotes/2019-01-10-hugo.md [hunk 14]
```diff
@@ -0,0 +1,109 @@
...
+
+## More entry points to content
+
+On top of a few general pages, our website is the home of many [blog posts](/blog), [tech notes](/technotes) and [community calls](/commcalls). We've created and improved pages referencing content over all these categories, so that you might more easily bounce [from say one software peer review post to the next](/tags/software-peer-review), read [all posts by our community manager](/authors/stefanie-butland), or simply find [all the things we ever published](/archive).
+
+### Tags and authors pages
+
+We now have a [tags page](/tags) referencing all tags along with the number of posts associated to them. The tags of a blog post, tech note or comm call are shown at the bottom, so that after reading you can choose to check out more content on the topic. After tags became even more valuable, our community manager Stefanie Butland added many of them and for making that process smoother we've created a [Shiny app to update tags more easily over many posts, `blogyaml`](https://github.com/ropenscilabs/blogyaml). It's worked well for us, and if you notice something wrong, well it has an issue tracker!
```
Comment (stefaniebutland) [inline content/technotes/2019-01-10-hugo.md]:
```suggestion
We now have a [tags page](/tags) referencing all tags along with the number of posts associated to them. The tags of a blog post, tech note or community call are shown at the bottom, so that after reading you can choose to check out more content on the topic. After tags became even more valuable, our community manager Stefanie Butland added many of them and for making that process smoother we've created a [Shiny app to update tags more easily over many posts, `blogyaml`](https://github.com/ropenscilabs/blogyaml). It's worked well for us, and if you notice something wrong, well it has an issue tracker!
```

Comment (stefaniebutland) [reply content/technotes/2019-01-10-hugo.md]: diff not so visible - I just spelled out "community" in comm call here

File: content/technotes/2019-01-10-hugo.md [hunk 15]
```diff
@@ -0,0 +1,109 @@
...
+
+Other changes were adjustments to content such as YAML reformatting. Fear not, we rarely had to manually update metadata, we wrote scripts using the [`yaml` R package](https://CRAN.R-project.org/package=yaml). Find [such a script](https://github.com/ropensci/roweb2/issues/197#issuecomment-394264824).
+
+## More entry points to content
+
+On top of a few general pages, our website is the home of many [blog posts](/blog), [tech notes](/technotes) and [community calls](/commcalls). We've created and improved pages referencing content over all these categories, so that you might more easily bounce [from say one software peer review post to the next](/tags/software-peer-review), read [all posts by our community manager](/authors/stefanie-butland), or simply find [all the things we ever published](/archive).
```
Comment (stefaniebutland) [inline content/technotes/2019-01-10-hugo.md]:
```suggestion
On top of a few general pages, our website is the home of many [blog posts](/blog), [tech notes](/technotes) and [community calls](/commcalls). We've created and improved pages referencing content over all these categories, so that you might more easily bounce [from say, one software peer review post to the next](/tags/software-peer-review), read [all posts by our community manager](/authors/stefanie-butland), or simply find [all the things we ever published](/archive).
```

File: content/technotes/2019-01-10-hugo.md (hunk 14 above)
Comment (stefaniebutland) [reply content/technotes/2019-01-10-hugo.md]: I think it's quite good to leave me out of this para, esp since you linked to my posts above (thank you!). e.g. revise to "After tags became even more valuable, to make the process of adding or editing them smoother we created a ..."

File: content/technotes/2019-01-10-hugo.md [hunk 16]
```diff
@@ -0,0 +1,109 @@
...
+
+### Tags and authors pages
+
+We now have a [tags page](/tags) referencing all tags along with the number of posts associated to them. The tags of a blog post, tech note or comm call are shown at the bottom, so that after reading you can choose to check out more content on the topic. After tags became even more valuable, our community manager Stefanie Butland added many of them and for making that process smoother we've created a [Shiny app to update tags more easily over many posts, `blogyaml`](https://github.com/ropenscilabs/blogyaml). It's worked well for us, and if you notice something wrong, well it has an issue tracker!
+
+We also have added an [authors page](/authors) that's very similar. On each author's page, you'll find their title for rOpenSci staff members, their online home, possibly their Twitter handle, and all the content they've authored on our website. [See for instance Stefanie Butland's author page](authors/stefanie-butland). These pages are linked from the bottom of posts for returning authors. Thanks to [Angela Li](https://angela-li.github.io/) for [requesting we added authors pages](https://github.com/ropensci/roweb2/issues/365).
```
Comment (stefaniebutland) [inline content/technotes/2019-01-10-hugo.md]:
```suggestion
We also have added an [authors page](/authors) that's very similar. On each author's page, you'll find their title for rOpenSci staff members, their online home, possibly their Twitter handle, and all the content they've authored on our website. [See for instance Stefanie Butland's author page](authors/stefanie-butland). These pages are linked from the bottom of posts for returning authors. Thanks to [Angela Li](https://angela-li.github.io/) for [requesting we add authors pages](https://github.com/ropensci/roweb2/issues/365).
```

Comment (stefaniebutland) [inline content/technotes/2019-01-10-hugo.md]: " authors page that's very similar" ...to what exactly?

Comment (stefaniebutland) [inline content/technotes/2019-01-10-hugo.md]: Here it would be nice to link to a community member's author page, like `[See for instance Nick Tierney's author page](authors/nicholas-tierney/)`

File: content/technotes/2019-01-10-hugo.md [hunk 17]
```diff
@@ -0,0 +1,109 @@
...
+
+Technically, getting tags and authors page and tag pages is quite straightforward with Hugo thanks to _taxonomies_. To switch them on, we added these lines to the website config.toml
```
Comment (stefaniebutland) [inline content/technotes/2019-01-10-hugo.md]:
```suggestion
Technically, getting tags, authors pages and tag pages is quite straightforward with Hugo thanks to _taxonomies_. To switch them on, we added these lines to the website config.toml
```

File: content/technotes/2019-01-10-hugo.md [hunk 18]
```diff
@@ -0,0 +1,109 @@
...
+  author = "authors"
+```
+
+We added this option as well
+
+```
+preserveTaxonomyNames = true
+```
+
+to not mess up with authors' names capitalization when listing them.
```
Comment (stefaniebutland) [inline content/technotes/2019-01-10-hugo.md]:
```suggestion
to not mess up authors' names capitalization when listing them.
```

File: content/technotes/2019-01-10-hugo.md [hunk 19]
```diff
@@ -0,0 +1,109 @@
...
+
+For authors, a challenge was moving metadata around, from storing authors' metadata (online home, Twitter username) inside posts metadata to having authors metadata files under content/authors, see [this example](https://github.com/ropensci/roweb2/blob/master/content/authors/ignasi-bartomeus/_index.md), but as mentioned earlier, with the `yaml` package you can edit hundreds of Markdown files at once, granted you check the edits carefully.
```
Comment (stefaniebutland) [inline content/technotes/2019-01-10-hugo.md]: sentence is confusing with "metadata" used 4 times

File: content/technotes/2019-01-10-hugo.md [hunk 20]
```diff
@@ -0,0 +1,109 @@
...
+
+Read more about [Hugo taxonomies](https://gohugo.io/content-management/taxonomies/), how to write [templates for taxonomy pages](https://gohugo.io/templates/taxonomy-templates/) and [where to save them so Hugo finds them](https://gohugo.io/templates/lookup-order/#examples-layout-lookup-for-taxonomy-list-pages). In our case, the tags and authors pages have different templates: [tags page](https://github.com/ropensci/roweb2/blob/master/themes/ropensci/layouts/_default/terms.html), [tag pages](https://github.com/ropensci/roweb2/blob/master/themes/ropensci/layouts/taxonomy/tag.html), [authors templates](https://github.com/ropensci/roweb2/tree/master/themes/ropensci/layouts/authors)
```
Comment (stefaniebutland) [inline content/technotes/2019-01-10-hugo.md]: might need a few more explanatory words since a reader won't know difference between "tags page" and "tag pages" without clicking. And with so many links they might not click

File: content/technotes/2019-01-10-hugo.md [hunk 21]
```diff
@@ -0,0 +1,109 @@
...
+
+### Archive
+
+With precious content scattered over three categories (blog, tech notes, community calls), getting an overview can be tricky hence our adding an [archive page](/archive) featuring all our website content [from our very first blog post](https://ropensci.org/blog/2012/11/26/is-invasive/). Behind the scenes, we [list pages with a date](https://github.com/ropensci/roweb2/blob/master/themes/ropensci/layouts/_default/single.html#L29) and [feeds them to an archive partial](https://github.com/ropensci/roweb2/blob/master/themes/ropensci/layouts/partials/archive.html) that we use for the [technotes page](/technotes) and authors pages as well for consistent presentation.
```
Comment (stefaniebutland) [inline content/technotes/2019-01-10-hugo.md]:
```suggestion
With precious content scattered over three categories (blog, tech notes, community calls), getting an overview can be tricky, hence our adding an [archive page](/archive) featuring all our website content [from our very first blog post](https://ropensci.org/blog/2012/11/26/is-invasive/). Behind the scenes, we [list pages with a date](https://github.com/ropensci/roweb2/blob/master/themes/ropensci/layouts/_default/single.html#L29) and [feed them to an archive partial](https://github.com/ropensci/roweb2/blob/master/themes/ropensci/layouts/partials/archive.html) that we use for the [tech notes page](/technotes) and authors pages as well for consistent presentation.
```

Comment (stefaniebutland) [reply content/technotes/2019-01-10-hugo.md]: suggested commit = 1) adding a comma to "tricky, hence"; 2) feeds -> feed; 3) technotes -> tech notes

Comment (stefaniebutland) [inline content/technotes/2019-01-10-hugo.md]: I think for less experienced readers, saying "we list pages with a date" may be confusing

File: content/technotes/2019-01-10-hugo.md [hunk 22]
```diff
@@ -0,0 +1,109 @@
...
+
+And should you ever get lost, we've now got a [cool 404 page](/404), although mentioning it probably lessens its coolness. [Here is its source](https://github.com/ropensci/roweb2/blob/master/themes/ropensci/layouts/404.html).
+
+## More content
+
+In the previous section we mention the [community calls living in ropensci.org](/commcalls). They used to have their own source and subdomain, but we've transferred them which has several advantages:
+
+* When reading about community calls, you're looking at the same design than when reading tech notes which makes us happy since we like our website. Besides, thanks to the footer and header, you're more prone to bouncing on over parts of the website. The [community calls list](/commcalls) list and [individual pages](https://ropensci.org/commcalls/2018-12-18/) use partials from the rest of the website, and our Hugo experience. In each community call page, the video recording is for instance embedded!
```
Comment (stefaniebutland) [inline content/technotes/2019-01-10-hugo.md]:
```suggestion
* When reading about community calls, you're looking at the same design as when reading tech notes which makes us happy since we like our website. Besides, thanks to the footer and header, you're more prone to bouncing on over parts of the website. The [community calls list](/commcalls) and [individual pages](https://ropensci.org/commcalls/2018-12-18/) use partials from the rest of the website, and our Hugo experience. In each community call page, the video recording is for instance embedded!
```

Comment (stefaniebutland) [inline content/technotes/2019-01-10-hugo.md]: "thanks to the footer and header, you’re more prone to bouncing on over parts of the website" could use some clarification. Do you mean "thanks to the footer and header, you’re more likely to bounce over to another part of the website, rather than leave" ??

Comment (stefaniebutland) [inline content/technotes/2019-01-10-hugo.md]: not sure what you mean by "and our Hugo experience"

Comment (stefaniebutland) [inline content/technotes/2019-01-10-hugo.md]: "In each community call page, the video recording is for instance embedded!"
the phrase "for instance" should be moved earlier or later in sentence.

I'm not clear on how this sentence relates to partials or Hugo experience.

My general comment about the two "More content" bullets is that I don't see a distinction between what you are trying to convey in each of them.

Comment (stefaniebutland) [inline content/technotes/2019-01-10-hugo.md]: I'm guessing that some readers are keen to learn about the technical side but that a term like "partials" would not be understood without some experience. Could you define on first use, and/or link to an explanation?

File: content/technotes/2019-01-10-hugo.md [hunk 23]
```diff
@@ -0,0 +1,109 @@
...
+
+* Community calls are now more similar to blog posts and tech notes, and are cross-listed via their tags and authors so that they're more easy to find.
+
+## Feed updates
+
+If you want to keep up with our production, you can subscribe to different feeds (or [our newsletter](https://news.ropensci.org/) since it includes mentions of recent posts!).
+
+### All the feeds
+
+Our website has both RSS and JSON feeds. We added a [JSON feed](https://jsonfeed.org/), feed format introduced in May last year, because we could and because some people might like that.
```
Comment (stefaniebutland) [inline content/technotes/2019-01-10-hugo.md]: "JSON feed, feed format" doesn't read clearly. Can you omit a "feed" or is it meant to be that way?

File: content/technotes/2019-01-10-hugo.md [hunk 24]
```diff
@@ -0,0 +1,109 @@
...
+
+## Twitter friendliness
+
+While Twitter is not the only platform on which our content is shared, it is an important player so we've upped our Twitter game with two edits.
+
+{{< tweet 1073519094958313472>}}
+
+We've worked on adding Twitter metadata to our [head partial](https://github.com/ropensci/roweb2/blob/master/themes/ropensci/layouts/partials/head.html) so when tweeting links from the website you get a pretty card (see above). We've used [resources listed in this GitHub issue](https://github.com/ropensci/roweb2/issues/290#issue-367542204) and [Twitter card validator](https://cards-dev.twitter.com/validator) on Netlify preview links. Whilst updating the head partial we realized not all our pages used it so we homogeneized all pages. Some of our pages like [about](/about) have fancy stuff that's not Markdown based, but we now handle that with partials listed in YAML and then called from the [single.html default template](https://github.com/ropensci/roweb2/blob/master/themes/ropensci/layouts/_default/single.html). Here [the source of our about page](https://github.com/ropensci/roweb2/blob/master/content/about.md).
```
Comment (stefaniebutland) [inline content/technotes/2019-01-10-hugo.md]:
```suggestion
We've worked on adding Twitter metadata to our [head partial](https://github.com/ropensci/roweb2/blob/master/themes/ropensci/layouts/partials/head.html) so when tweeting links from the website you get a pretty card (see above). We've used [resources listed in this GitHub issue](https://github.com/ropensci/roweb2/issues/290#issue-367542204) and [Twitter card validator](https://cards-dev.twitter.com/validator) on Netlify preview links. Whilst updating the head partial we realized not all our pages used it so we homogeneized all pages. Some of our pages like [about](/about) have fancy stuff that's not Markdown based, but we now handle that with partials listed in YAML and then called from the [single.html default template](https://github.com/ropensci/roweb2/blob/master/themes/ropensci/layouts/_default/single.html). Here is [the source of our about page](https://github.com/ropensci/roweb2/blob/master/content/about.md).
```

Comment (stefaniebutland) [inline content/technotes/2019-01-10-hugo.md]: should you acknowledge Marie-Helene Burle for twitter card suggestion since you acknowledged Angela? https://discuss.ropensci.org/t/suggestion-improve-visual-appeal-of-ropensci-blog-pages-and-make-them-easy-to-post-on-twitter-with-an-appealing-image/1391/2

File: content/technotes/2019-01-10-hugo.md [hunk 25]
```diff
@@ -0,0 +1,109 @@
...
+
+{{< tweet 953090299819118592 >}}
+
+We _put a bird on it_, in Mara Averick's words, i.e. we now add a Twitter icon near authors' names when their metadata contains a Twitter username (have you noticed the bird near my name at the top of this post?). That happens [in the authors.html partial](https://github.com/ropensci/roweb2/blob/master/themes/ropensci/layouts/partials/authors.html). Authors pages also link to Twitter accounts. By the way if you're ever authored a post for us feel free to open a pull request updating your metadata, e.g. if you were Ignasi Bartomeus you'd update [this file](https://github.com/ropensci/roweb2/blob/master/content/authors/ignasi-bartomeus/_index.md).
```
Comment (stefaniebutland) [inline content/technotes/2019-01-10-hugo.md]:
```suggestion
We _put a bird on it_, in Mara Averick's words, i.e. we now add a Twitter icon near authors' names when their metadata contains a Twitter username (have you noticed the bird near my name at the top of this post?). That happens [in the authors.html partial](https://github.com/ropensci/roweb2/blob/master/themes/ropensci/layouts/partials/authors.html). Authors pages also link to Twitter accounts. By the way, if you're ever authored a post for us feel free to open a pull request updating your metadata, e.g. if you were Ignasi Bartomeus you'd update [this file](https://github.com/ropensci/roweb2/blob/master/content/authors/ignasi-bartomeus/_index.md).
```

Comment (stefaniebutland) [reply content/technotes/2019-01-10-hugo.md]: I'm not sure why the diff sometimes doesn't show suggested changes for commit. let me know if in future I should note what suggested changes are made

File: content/technotes/2019-01-10-hugo.md [hunk 26]
```diff
@@ -0,0 +1,109 @@
...
+
+## A calendar automatically up-to-date
+
+Have you noticed [our calendar of events](/events)? It has its [own page](/events) but is also displayed from several other places of the website. We used to have a clunky calendar update workflow. Now, events information are stored as YAML in content/events, see [this example](https://github.com/ropensci/roweb2/blob/master/content/events/2019-01-15-rstudioconf.md), and the [calendar partial](https://github.com/ropensci/roweb2/blob/master/themes/ropensci/layouts/partials/calendar.html) creates an HTML table from all events and community calls between _now_ (Hugo `now` function) and one year from now. We [trigger one website build per day thanks to a Zapier Netlify integration](https://zapier.com/apps/netlify/integrations).
```
Comment (stefaniebutland) [inline content/technotes/2019-01-10-hugo.md]:
```suggestion
Have you noticed [our calendar of events](/events)? It has its [own page](/events) but is also displayed from several other places on the website. We used to have a clunky calendar update workflow. Now, events information are stored as YAML in content/events, see [this example](https://github.com/ropensci/roweb2/blob/master/content/events/2019-01-15-rstudioconf.md), and the [calendar partial](https://github.com/ropensci/roweb2/blob/master/themes/ropensci/layouts/partials/calendar.html) creates an HTML table from all events and community calls between _now_ (Hugo `now` function) and one year from now. We [trigger one website build per day thanks to a Zapier Netlify integration](https://zapier.com/apps/netlify/integrations). 
```

File: content/technotes/2019-01-10-hugo.md [hunk 27]
```diff
@@ -0,0 +1,109 @@
...
+
+Each event also gets its own page which allows displaying more information than in the calendar table, e.g. read more about [what Karthik Ram and Jeroen Ooms will be up to at rstudio::conf](https://ropensci.org/events/2019-01-15-rstudioconf/).
+
+## Conclusion
+
+In the last months, our website got more pages, list of pages and cross-references, JSON feeds, and Twitter metadata. It should have gotten nicer for you to browse and we hope you'll never leave it! We're open to more suggestions of improvements, in the comments below or in [roweb2 issue tracker](https://github.com/ropensci/roweb2/).
```
Comment (stefaniebutland) [inline content/technotes/2019-01-10-hugo.md]:
```suggestion
In the last months, our website has gained more pages, lists of pages and cross-references, JSON feeds, and Twitter metadata. It should have gotten nicer for you to browse and we hope you'll never leave it! We're open to more suggestions of improvements, in the comments below or in [roweb2 issue tracker](https://github.com/ropensci/roweb2/).
```

File: content/technotes/2019-01-10-hugo.md (hunk 16 above)
Comment (stefaniebutland) [reply content/technotes/2019-01-10-hugo.md]: or to Kelly O'Briant or even both to feature different aspects

File: content/technotes/2019-01-10-hugo.md (hunk 24 above)
Comment (maelle) [reply content/technotes/2019-01-10-hugo.md]: Good point!

File: content/technotes/2019-01-10-hugo.md (hunk 25 above)
Comment (maelle) [reply content/technotes/2019-01-10-hugo.md]: it's fine, _I_ should remember to use all your suggestions before making any other change, because now they are outdated and I need to add them manually :woman_facepalming:

File: content/technotes/2019-01-10-hugo.md (hunk 26 above)
Comment (maelle) [reply content/technotes/2019-01-10-hugo.md]: I don't see the edit :-(
